contracts/Bridge/XVSProxyOFTSrc.sol

```diff
@@ -53,6 +58,23 @@ contract XVSProxyOFTSrc is BaseXVSProxyOFT {
         emit FallbackWithdraw(to_, amount_);
     }
 
+    /**
+     * @notice Forces the lock of tokens by increasing outbound amount and transferring tokens from the sender to the contract.
+     * @param amount_ The amount of tokens to lock.
+     * @custom:access Only owner.
+     * @custom:event Emits FallbackDeposit, once done with transfer.
+     */
+    function fallbackDeposit(uint256 amount_) external onlyOwner {
+        (uint256 actualAmount, ) = _removeDust(amount_);
```

Comment:
Do we need to remove dust? XVS doesn't have fees

Reply:
This operation is perform before debit in LayerZero because of shared decimals see [here](https://github.com/LayerZero-Labs/solidity-examples/blob/main/contracts/token/oft/v2/OFTCoreV2.sol#L125 ).